### PR TITLE
docs: add selin tunc to contributors

### PR DESCRIPTION
**Pull request by celinetunc**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -783,6 +783,14 @@
     "bio": "Santosh is a GDE for Angular and loves to contribute to Open Source. He is the creator of ng deploy for netlify and core team member for NestJS Addons. He writes for AngularInDepth, mentors for DotNetTricks, organizes Pune Tech Meetup, and conducts free workshops on Angular.",
     "groups": ["GDE"]
   },
+  "selintunc": {
+      "name": "Selin Tunc",
+      "picture": "selin.jpeg",
+      "twitter": "selintunc_",
+      "website": "https://www.linkedin.com/in/selinkaragulletunc",
+      "bio": "Selin is a Software Developer Expert using Angular and Java at Garanti BBVA Technology in Turkey. She is the first woman Angular GDE in Turkey. She spends her spare time with her family, especially her little son, and by keeping herself up-to-date.
```
Comment (jessicajaniuk) [inline aio/content/marketing/contributors.json:791]:
```suggestion
      "bio": "Selin is a Software Developer Expert using Angular and Java at Garanti BBVA Technology in Turkey. She is the first woman Angular GDE in Turkey. She spends her spare time with her family, especially her little son, and by keeping herself up-to-date.",
```

**Review by jessicajaniuk — CHANGES_REQUESTED** [review]
You're missing closing quotes and a comma. Once you fix that, this should be green and ready to merge.

**Review by jessicajaniuk — DISMISSED** [review]
LGTM :cookie:

Thanks for the quick fix!

File: aio/content/marketing/contributors.json [hunk 2]
```diff
@@ -783,6 +783,14 @@
...
     "groups": ["GDE"]
   },
+  "selintunc": {
+      "name": "Selin Tunc",
+      "picture": "selin.jpeg",
+      "twitter": "selintunc_",
+      "website": "https://www.linkedin.com/in/selinkaragulletunc",
+      "bio": "Selin is a Software Developer Expert using Angular and Java at Garanti BBVA Technology in Turkey. She is the first woman Angular GDE in Turkey. She spends her spare time with her family, especially her little son, and by keeping herself up-to-date.",
+      "groups": ["GDE"]
+},
```
Comment (mgechev) [inline aio/content/marketing/contributors.json:793]:
```suggestion
  },
```

**Review by mgechev — DISMISSED** [review]

**Review by jessicajaniuk — DISMISSED** [review]

**Comment by jessicajaniuk** [comment]
@celinetunc  The size change here is not from your PR. If you rebase to main, this should go away. Could you also squash your commits? That should eliminate the lint issue.

**Review by mgechev — DISMISSED** [review]

**Review by jessicajaniuk — APPROVED** [review]

**Review by mgechev — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 3e873b55165cabb8d340811670a159e6ad9c3680.

**Comment by celinetunc** [comment]
> @celinetunc The size change here is not from your PR. If you rebase to main, this should go away. Could you also squash your commits? That should eliminate the lint issue.

Thank you so much for your support Jessica. :)

**Pull request closed without merging**